### DISCUSSION TITLE
Is there a limit at the number of collection files where using Statamic is not a good idea?

### DISCUSSION THREAD

**Discussion by smayzes** [statamic/cms#6494] [Troubleshooting] [answered]
I was investigating rebuilding one of my apps using Statamic for the CMS layer (Am currently using Nova).

I made a command that created several thousand (7k+) .md files from my DB records, I was assuming/hoping that the app could run this many, however in hindsight it's probably not a good idea with that many. 

Has anyone had luck scaling their collections?

**Comment by duncanmcclean** [comment, marked as answer]
I think you might be better keeping those records in the database and using an addon like [Runway](https://statamic.com/addons/double-three-digital/runway) to let you manage them within the Statamic CP (disclaimer: I'm the dev behind Runway).
